### DISCUSSION TITLE
`input --suppress-output` doesn't work on Windows

### DISCUSSION THREAD

**Discussion by colorlessred** [nushell/nushell#9018] [Q&A]
without `--suppress-output` it works as expected
```
> echo (input "pwd: ")
pwd: abc
abc
```

with the flag, it exists immediately (without asking for any input)

```
> echo (input "pwd: " -s)
pwd:

```

this is nushell 0.79.0 running on windows
```
> $nu.os-info | to text
name: windows
arch: x86_64
family: windows
kernel_version: 22621
```

**Comment by fdncred** [comment]
To suppress the output, I think you need to do it like this. It's assuming you're going to assign the output to a variable, I believe. I just tested on MacOS and it worked as expected in v0.79.1.
```
let userinput = (input -s)
```

**Reply by colorlessred** [reply]
just tried that exact line on windows and it exits immediately
```
~
> let userinput = (input -s)

~
>
```

**Reply by fdncred** [reply]
Weird. On MacOS, it waits for me to type, doesn't show anything, but when I hit enter, it stops and whatever I typed is in the $userinput variable.

**Reply by colorlessred** [reply]
so it looks like a bug on the windows version. Clearly it doesn't wait for any input for me. Maybe windows end-of-line passed to `input` as the "enter"?

```
❯ echo $'got: (input "pwd: ")'
pwd: wdb
got: wdb

> echo $'got: (input "pwd: " -s)'
pwd: got:

>
```

**Comment by colorlessred** [comment]
please notice that this works on Window 

```
❯ echo $'got: (input "pwd: " -s)'
pwd: got: abc

~ took 2s
```

with nushell 0.78.0. 

I have then updated nushell to 0.79 and it stopped working. So I think this is a regression from 0.78 to 0.79.

**Reply by fdncred** [reply]
Looks like it. I'm guessing you're right and it's a line ending problem. It would be good to know which PR broken it.
